storage/api/Storage.Samples/StorageControlDeleteManagedFolder.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_control_managed_folder_delete]
+using Google.Cloud.Storage.Control.V2;
+using System;
+
+public class StorageControlDeleteManagedFolderSample
+{
+    public void StorageControlDeleteManagedFolder(string bucketName = "your-unique-bucket-name",
+        string managedFolderId = "your_managed_folder_id")
+    {
+        StorageControlClient storageControl = StorageControlClient.Create();
+
+        string managedFolderResourceName =
```

Comment:
I think it would be better to just create a new ManagedFolderName and call DeleteManagedFolder(ManagedFolderName). IMO we should avoid the Format calls as far as possible, using the resource name classes.